### DISCUSSION TITLE
I second that

### DISCUSSION THREAD

**Discussion by nmbr73** [nmbr73/Shadertoys#48] [General]
This is to provide a more recent Atom package (Shadertoys v1.2) for the Reactor. The 1.0 release we did in June 2021 (time flies) contained 50 toys. This one should contain 95 toys running on Mac and Windows (on Windows it should even be 157 toys) - thanks to @J-i-P-i for all the conversions! Please note that OpenCL compatibility has been dropped for Mac - use the Fuses on Metal only! Additionally this Release adds a browser to the Scripts menu, that allows you to add a shader to your composition - thanks @AndrewHazelden for the great support!

<hr /><em>This discussion was created from the release <a href='https://github.com/nmbr73/Shaderfuse/releases/tag/v1.2-RC1'>I second that</a>.</em>

**Comment by rne1223** [comment]
Amazing job guys 👏👏👏. You are all an inspiration to follow.

**Reply by nmbr73** [reply]
Hey! Long time no see! I am very happy to hear from you - I hope you are doing well?!?
